### ISSUE TITLE
Custom test run title not working

### ISSUE THREAD

**Issue by vtbassmatt** [tonybaloney/pytest-azurepipelines#10]
Awesome work, thank you! (Disclosure: I'm a PM on Azure Pipelines, and a huge fan of Python :grin:)

I can't seem to get `--test-run-title` working on 0.3.0. Snippet from my YAML:
```yaml
  - script: |
      pip install pytest pytest-azurepipelines
      pytest tests --doctest-modules --test-run-title="Foo"
```

Results in UI:
![image](https://user-images.githubusercontent.com/714283/51849880-9e9a8b80-22ee-11e9-8a51-691ae24163e3.png)

I noticed your pipeline is [having the same issue](https://dev.azure.com/AnthonyShaw/pytest-azurepipelines/_build/results?buildId=183&view=ms.vss-test-web.test-result-details).

**Comment by vtbassmatt** [comment]
I'm not sure. Let me ask some colleagues tomorrow.

**Comment by vtbassmatt** [comment]
Looks like it's [`runTitle`](https://github.com/Microsoft/azure-pipelines-tasks/blob/324a8d819023ad7aa46c6bc7ca62c3f59c7a7a17/Tasks/PublishTestResultsV2/publishtestresults.ts#L30)

**Comment by tonybaloney** [comment]
that's what it is already ;-) guess I must be doing something wrong..

**Comment by tonybaloney** [comment]
@vtbassmatt fixed in 0.4.0

**Comment by tonybaloney** [comment]
I'd like to repro #6 also beforehand @vtbassmatt
